api/src/main/java/org/openmrs/api/VisitService.java

```diff
@@ -258,6 +259,15 @@ public List<Visit> getVisits(Collection<VisitType> visitTypes, Collection<Patien
 	        Date minEndDatetime, Date maxEndDatetime, Map<VisitAttributeType, Object> attributeValues,
 	        boolean includeInactive, boolean includeVoided) throws APIException;
 	
+	/**
+	 * Gets the visits matching the specified search criteria
+	 * 
+ 	 * @param visitSearchCriteria
+	 * @return
+	 * @throws APIException
+	 */	
```

Comment:
https://openmrs.atlassian.net/wiki/spaces/docs/pages/25477044/Java+Conventions#New-Public-Methods-and-Classes